src/snovault/elasticsearch/indexer.py

```diff
@@ -46,118 +302,146 @@ def index(request):
     recovery = request.json.get('recovery', False)
     es = request.registry[ELASTIC_SEARCH]
     indexer = request.registry[INDEXER]
-
+    stage_for_followup = request.registry.settings.get("stage_for_followup", False)
```

Comment:
What is this supposed to be for??  There are no tests, no config...

Reply:
in encoded stage_for_followup is set, else secondary indexer never runs.  encoded test_indexer.py failed because the app didn't have that setting and I had to add it.  So this is tested in encoded.